docs/UserGuide.md

```diff
@@ -100,180 +93,245 @@ Do have a look at our Table of Contents to locate the help you need!
 
 </div>
 
---------------------------------------------------------------------------------------------------------------------
+As you explore the features that SociaLite has to offer, look out for these boxes:
 
-## Basic Functionality <a name="basic_functionality"></a>
+This box tells you the acceptable fields that you can provide for a particular command:
 
-### Adding a person: `add`
+<div markdown="block" class="alert alert-primary">
+**:mag_right: Format:**
+`add n/NAME p/PHONE_NUMBER [t/TAG]…​ [date/NAME:YYYY-MM-DD[:monthly|:yearly]]…​ [fb/FACEBOOK] [ig/INSTAGRAM] [tele/TELEGRAM] [tiktok/TIKTOK] [twitter/TWITTER]`
+* Detail 1
+* Detail 2
+* Detail 3
+</div>
 
-Adds a person to the address book.
+If you still aren't sure of how a particular command works, this box shows you examples that you can try out:
```

Comment:
I think a better way might be "blue boxes shows..." and "green boxes shows...", since you have already painstakingly (thanks for the overhaul by the way!) to put them into colour-coded HTML boxes!

Reply:
Suggestion adopted in line 101 & 111 of latest commit